specification/applicationinsights/resource-manager/readme.typescript.md

```diff
@@ -7,6 +7,6 @@ Please also specify `--typescript-sdks-folder=<path to root folder of your azure
 typescript:
   azure-arm: true
   package-name: "@azure/arm-appinsights"
-  output-folder: "$(typescript-sdks-folder)/packages/@azure/arm-appinsights"
+  output-folder: "$(typescript-sdks-folder)/sdk/appinsights/arm-appinsights"
```

Comment:
We should try and keep the folder names the same as they are in the specification repo so this should be "applicationinsights" instead of "appinsights".

Reply:
Done